OpenRA.Mods.Common/Widgets/Logic/Ingame/ClassicProductionLogic.cs

```diff
@@ -29,7 +29,7 @@ void SetupProductionGroupButton(OrderManager orderManager, ProductionTypeButtonW
 
 			// Classic production queues are initialized at game start, and then never change.
 			var queues = world.LocalPlayer.PlayerActor.TraitsImplementing<ProductionQueue>()
-				.Where(q => q.Info.Type == button.ProductionGroup)
+				.Where(q => (q.Info.Group != null ? q.Info.Group : q.Info.Type) == button.ProductionGroup)
```

Comment:
This can simplify to `q => (q.Info.Group ?? q.Info.Type) == button.ProductionGroup`